### PR TITLE
[Proposal] Put debugger stack on the left

### PR DESCRIPTION
**Pull request by guillep**
imagen

### CODE REVIEW

**Comment by estebanlm** [comment]
not against :)

but I am working on the message browser, and I added a "flip" button to allow the user to switch as they please.
I guess it could be the same for the debugger? 
 
EDIT: submitted before finish, fixed :P

**Comment by StevenCostiou** [comment]
Following this proposition, I planned to propose a dynamic switch between layouts in the debugger.
Not exactly a flip, more there is a list of layouts some provided by default and if you want you can add yours, and then you select the one that you want in the debugger (from a menu somewhere) and it becomes the new default in your image.

I wanted to do that last friday because it is really trivial to do, but I got sick with kids...

I'll try it tomorrow if I don't have kids on my hands :p

File: src/NewTools-Debugger/StDebugger.class.st [hunk 1]
```diff
@@ -1147,7 +1138,7 @@ StDebugger >> stack [
 { #category : 'specs' }
 StDebugger >> stackAndCodeLayout [
 
-	^ SpPanedLayout newTopToBottom
+	^ SpPanedLayout newLeftToRight
 		positionOfSlider: 30 percent;
 		add: self stackLayout;
 		add: self codeLayout;
```
Comment (StevenCostiou) [inline src/NewTools-Debugger/StDebugger.class.st:1144]: This is actually the only change right?
All the other diffs are just code reformating? (hard to see)

**Review by StevenCostiou — APPROVED** [review]

File: src/NewTools-Debugger/StDebugger.class.st (hunk 1 above)
Comment (guillep) [reply src/NewTools-Debugger/StDebugger.class.st:1144]: I changed also the columns.
Having too many columns is good for a vertical layout, where the method list can be very large.
But for the horizontal layout the list should be narrower, so having too many columns is not very good there...
Tradeoffs :)

**Comment by StevenCostiou** [comment]
We can close it now, since another PR implementing this change has been merged.

**Pull request closed without merging**